tendrl/node_agent/manager/manager.py

```diff
@@ -227,6 +230,35 @@ def on_pull(self, raw_data):
                     node_id=raw_data["node_id"],
                 )
             )
+        if "disks" in raw_data:
+            LOG.info("on_pull, Updating disks")
+            try:
+                self.etcd_client.delete(
```

Comment:
What is raw_disks passed is None? Do I feel we should delete the entry from etcd post some initial checks only.